src/Configuration/src/Encryption/DecryptionConfigurationProvider.cs

```diff
@@ -14,6 +14,7 @@ internal sealed partial class DecryptionConfigurationProvider(
     : CompositeConfigurationProvider(providers, loggerFactory)
 {
     private readonly ILogger<DecryptionConfigurationProvider> _logger = loggerFactory.CreateLogger<DecryptionConfigurationProvider>();
+    private ITextDecryptor? _textDecryptor = textDecryptor;
```

Comment:
What is the significance of this change?

Reply:
It's adapting to a change in how Resharper rules work, see https://youtrack.jetbrains.com/issue/RSRP-498478/Breaking-primary-constructors-with-readonly-field#focus=Comments-27-10740044.0-0 for details.

Because the field is overwritten, it should be extracted to a field.